tests-behat/bootstrap/Context.php

```diff
@@ -500,7 +500,7 @@ public function iCheckIfWordMatch()
 
         $responseWord = $wordResponseContainer->getText();
 
-        if (preg_replace('/[^A-Za-z0-9\-]/', '', $expectedWord) !== preg_replace('/[^A-Za-z0-9\-]/', '', $responseWord)) {
+        if (preg_replace('/\s*/m', '', $expectedWord) !== preg_replace('/\s*/m', '', $responseWord)) {
```

Comment:
```suggestion
        if (preg_replace('~\s*~', '', $expectedWord) !== preg_replace('~\s*~', '', $responseWord)) {
```

I highly prefer `~` as regex delimiter. `m` flag completely not needed as `\s` match NL even then.

Reply:
ok - will try.